SportHub.Data/Entities/ArticleForAutocomplete.cs

```diff
@@ -0,0 +1,7 @@
+namespace SportHub.Data.Entities;
+
+public class ArticleForAutocomplete
```

Comment:
I don't think it's a good idea to link it to Autocomplete because backend should not depend on frontend